openhtf/util/monitors.py

```diff
@@ -108,7 +108,7 @@ def Wrapper(phase_func):
     @plugs.plug(update_kwargs=False, **monitor_plugs)
     @measurements.measures(
         measurements.Measurement(measurement_name).WithUnits(
-            units).WithDimensions(uom.UOM['MILLISECOND']))
+            units).WithDimensions(uom.MILLISECOND))
```

Comment:
@grybmadsci, did you choose `MILLISECOND` because `SECOND` wasn't in the units list before? Should we switch to seconds now?